src/Application.Models/Requests/CreateCustomerRequest.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+
+namespace Application.Models;
+public class CreateCustomerRequest
+{
+    public CreateCustomerRequest(
+        string fullName,
+        string email,
+        string emailConfirmation,
+        string cpf,
+        string cellphone,
+        DateTime birthdate,
+        bool emailSms,
+        bool whatsapp,
+        string country,
+        string city,
+        string postalCode,
+        string address,
+        int number
+    )
+    {
+        this.FullName = fullName;
+        this.Email = email;
+        this.EmailConfirmation = emailConfirmation;
+        this.Cpf = cpf;
+        this.Cellphone = cellphone;
+        this.Birthdate = birthdate;
+        this.EmailSms = emailSms;
+        this.Whatsapp = whatsapp;
+        this.Country = country;
+        this.City = city;
+        this.PostalCode = postalCode;
+        this.Address = address;
+        this.Number = number;
```

Comment:
O `this` é desnecessário
```suggestion
        FullName = fullName;
        Email = email;
        EmailConfirmation = emailConfirmation;
        Cpf = cpf;
        Cellphone = cellphone;
        Birthdate = birthdate;
        EmailSms = emailSms;
        Whatsapp = whatsapp;
        Country = country;
        City = city;
        PostalCode = postalCode;
        Address = address;
        Number = number;
```